### PR TITLE
Sport Schema v1.0.1: Fixing datatype of properties containing SKOS Concepts

### PR DESCRIPTION
**Pull request by bquinn**
This was a bug in the original schema so the change doesn't need a Standards Committee vote.

Also updating version number to 1.0.1 throughout and updating dates.

### CODE REVIEW

File: ontologies/iptc-sport-ontology.ttl
Comment (paulwilton) [inline ontologies/iptc-sport-ontology.ttl:1]: the ObjectProperty changes look good
however line 245  range  sport:event should be sport:Event

**Review by paulwilton — CHANGES_REQUESTED** [review]
the ObjectProperty changes look good
however line 245 range sport:event should be sport:Event

File: ontologies/iptc-sport-ontology.ttl
Comment (bquinn) [reply ontologies/iptc-sport-ontology.ttl:1]: Fixed - thanks! I forgot that you had reported this already...

**Comment by bquinn** [comment]
We also decided to change the versioning approach: #186

**Pull request merged**